docs/content/SelectMenu.md

```diff
@@ -40,6 +40,42 @@ SelectMenu components get `COMMON` system props. Read our [System Props](/system
 | :- | :- | :-: | :- |
 | initialTab | String | | If using the `SelectMenu.Tabs` component, you can use this prop to change the tab shown on open. By default, the first tab will be used.
 
+## SelectMenu.MenuContext
+SelectMenu.MenuContext is a [context object](https://reactjs.org/docs/context.html#reactcreatecontext) that exposes some helpful state values to be used via [`React.useContext`](https://reactjs.org/docs/hooks-reference.html#usecontext) in consuming applications.  SelectMenu.MenuContext can only be used in components that are already wrapped in a `SelectMenu` as `SelectMenu` contains the [context provider](https://reactjs.org/docs/context.html#contextprovider).
+
+### Values and functions available on MenuContext
+| Name | Type | Description |
+| :- | :- | :- |
+| selectedTab | string | The currently selected tab |
+| setSelectedTab | function | Used to update the currently selected tab state |
+| open | boolean | State for open/closed status of the menu content  |
```

Comment:
Would "modal" be more clear than "content" here, since the component that is being open and closed is called `Modal`?
```suggestion
| open | boolean | State for open/closed status of the menu modal  |
```